packages/core/src/db/db.ts

```
@@ -0,0 +1,231 @@
+        if (!finalised) {
+          finalised = true;
+          client.release();
+        }
+      };
+
+      return {
+        commit: async () => {
+          try {
+            await client.query('COMMIT');
+          } catch (error) {
+            throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the catch clause.


(lint/complexity/noUselessCatch)